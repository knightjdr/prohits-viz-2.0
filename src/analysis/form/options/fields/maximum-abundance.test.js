import React from 'react';
import { shallow } from 'enzyme';

import Maximumbundance from './maximum-abundance';

describe('Maximumbundance', () => {
  test('Renders', () => {
    const wrapper = shallow(
      <Maximumbundance
        analysisType="dotplot"
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
