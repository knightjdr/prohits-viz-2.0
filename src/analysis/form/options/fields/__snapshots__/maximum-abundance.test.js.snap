// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maximumbundance Renders 1`] = `
<CustomField
  helpMessage="Any readouts with an abundance value (e.g. spectral count) above this cutoff
      will be capped at this maximum in the output images. This is to give greater
      visual dynamic range for lower values when outlier readouts with very high
      abundance are present. This cutoff will be dependent on the instrument and
      interaction method used and should be selected based on the data set."
  inputType="number"
  label="Maximum abundance"
  name="abundanceCap"
  onChange={[Function]}
  placeHolder="Maximum abundance..."
  type="input"
/>
`;
